### ISSUE TITLE
Creating a Stripe account

### ISSUE THREAD

**Issue by KimPong** [volunteer-manage/Bears-Gone-Bush#2]
Hey @jarvisjohnson when you get a chance, could you please describe how I set up the Stripe account in the portal?

Many thanks xx

**Comment by jarvisjohnson** [comment]
Yep this is done automatically when you create the festival - I will send
you the stripe info.


+1 347 641 7815
skype: jarvisjohnson
github <http://github.com/jarvisjohnson> | linkedin
<https://www.linkedin.com/in/jarvis-johnson-65813453/>

**Comment by KimPong** [comment]
@jarvisjohnson thank you!
